### ISSUE TITLE
New syntax to ACL and nodes permission

### ISSUE THREAD

**Issue by tiago-cruz-movile** [chef/knife-acl#27]
Hello,

I used to use some like
```
$ knife acl add nodes mynode delete client mynode
```

But now, the message returns

```
FATAL: ERROR: To enforce best practice, knife-acl can only add a group to an ACL.
FATAL:        See the knife-acl README for more information.
```

Could you please tell me the best way to grant someone permission to one specific node without use groups? Or, if we don't have a group to use?

Thanks!

**Comment by jeremiahsnapp** [comment]
Hello @tiago-cruz-movile,
Thanks for pointing this out. I just released beta.2 with the ability to add clients to permissions. The following command should work now. Please let me know if you still have problems with it.

```
knife acl add client mynode nodes mynode delete
```

**Comment by tiago-cruz-movile** [comment]
Hello @jeremiahsnapp,

Worked with a charm, thanks a lot! :+1:

**Comment by jeremiahsnapp** [comment]
That's great! I'm glad it's working for your now. By the way, I just released 1.0.0. It's identical to the beta you have now but at least it's an official release. :) And now you can just install it with `chef gem install knife-acl`.
